wordpress-mobile/WordPress-Android
# Manage Insights screen

Fixes #9464.

To test:

1. Go to Stats
2. Tap the Manage toolbar button
3. Try adding and removing the insights by tapping on the + or - button
4. Tap the Save button
5. Notice the Insights list is updated and data is refreshed
6. Go back to the management screen
7. Try reordering the added insights by drag & dropping of list items
8. Tap the Save button
9. Notice the Insights list is reordered accordingly


![image](https://user-images.githubusercontent.com/1522856/55331749-39048000-5494-11e9-8864-9d8c834c988c.png)

Update release notes:
